### ISSUE TITLE
Issue: `prefer-takeuntil` rule does not recognize configured alias options in rxjs-angular-x

### ISSUE THREAD

**Issue by examan** [JasonWeinzierl/eslint-plugin-rxjs-angular-x#31]
## Description
The `prefer-takeuntil` rule in `eslint-plugin-rxjs-angular-x` fails to recognize configured `alias` options (`untilDestroyed`, `untilCompleted`), even though they are properly set in the ESLint configuration. This worked correctly in the original `eslint-plugin-rxjs-angular` (cartant version), but breaks in the `-x` fork.

## Expected Behavior
When `alias: ['untilDestroyed', 'untilCompleted']` is configured, the rule should accept these as valid alternatives to `takeUntil`, preventing false positives for `subscribe` calls that use these aliases.

## Actual Behavior
ESLint reports errors like "Forbids calling `subscribe` without an accompanying `takeUntil`" even when `untilDestroyed` or `untilCompleted` is present in the observable chain, within the specified `checkDecorators` (e.g., `@Component`).

## Steps to Reproduce
1. Configure ESLint with:
   ```javascript
   'rxjs-angular-x/prefer-takeuntil': [
     'error',
     {
       alias: ['untilDestroyed', 'untilCompleted'],
       checkDecorators: ['Component', 'Directive', 'Pipe', 'Injectable'],
     },
   ],
   ```
2. Use code like:
   ```typescript
   import { untilDestroyed } from '@ngneat/until-destroy';
   import { untilCompleted } from '@lib/util/rx/operators/until-completed';

   @UntilDestroy()
   @Component({...})
   export class MyComponent {
     ngOnInit() {
       observable.pipe(untilDestroyed(this)).subscribe(...);  // Should pass, but errors
       anotherObservable.pipe(untilCompleted()).subscribe(...);  // Should pass, but errors
     }
   }
   ```
3. Run ESLint: Errors are reported despite aliases being used.

## Environment
- ESLint version: 9.39.3
- `eslint-plugin-rxjs-angular-x` version: 1.0.0

## Additional Context
- Original plugin (cartant/eslint-plugin-rxjs-angular) handles aliases correctly.
- The issue appears to stem from stricter AST matching in the `-x` version, which may not fully support custom operators like `untilCompleted` or `@ngneat/until-destroy`'s `untilDestroyed`.
- Related code snippets from the original vs. `-x` implementations show differences in alias validation logic.

Please investigate and fix the alias recognition to match the original plugin's behavior. If this is intentional, update the documentation to clarify supported aliases.
